bin/standalone-installer-unix

```diff
@@ -87,8 +87,8 @@ main() {
 
     log "Checking installation"
     local installed_version
-    if ! installed_version="$("$DESTINATION"/nextstrain --version)"; then
-        die "installation check failed: unable to run \`\"$DESTINATION\"/nextstrain --version\`; look for error message above."
+    if ! installed_version="$(nextstrain --version)"; then
+        die "installation check failed: unable to run \`nextstrain --version\`; look for error message above."
```

Comment:
The use of the full path was intentional, as it won't ever be misleading if the PATH lookup doesn't go as expected (e.g. because shell setup above didn't fail but didn't work). I don't think we should use a PATH lookup here.